util/find-rules-without-test/README.md

```diff
@@ -0,0 +1,31 @@
+# Find the rules without test cases
+
+This page describes how can you find the rules without any test cases
+
+## Goal
+
+The main goal that we must to have at least regression test for all relevant REQUEST-* rules. (In this context, the PL controll rules are not relevant, because they do not need to have tests.)
+
+You need to pass the CORERULESET root as argument.
```

Comment:
I think it's better to use the same term as below in line 11: CORERULESET_ROOT.

Does it make sense to add an example call (I'm not sure):

`util/find-rules-without-test/find_rules_without_test.py /path/to/coreruleset`

Reply:
Fixed.